packages/mobx/src/api/autorun.ts

```diff
@@ -86,14 +86,14 @@ export function autorun(
     return reaction.getDisposer_()
 }
 
-export type IReactionOptions = IAutorunOptions & {
+export type IReactionOptions<T> = IAutorunOptions & {
     fireImmediately?: boolean
-    equals?: IEqualsComparer<any>
+    equals?: IEqualsComparer<T>
 }
 
 const run = (f: Lambda) => f()
 
-function createSchedulerFromOptions(opts: IReactionOptions) {
+function createSchedulerFromOptions(opts: IAutorunOptions) {
```

Comment:
Why we need replace `IReactionOptions` with `IAutorunOptions`?

Reply:
Because it's also used in `autorun` above, and I had some trouble getting the generic type work properly here. This function doesn't care about anything outside `IAutorunOptions` anyway.

Reply:
@mweststrate Sir, Please code review here if you have time.

Reply:
/cc @urugator

Reply:
Sorry, I am not much into TS stuff, so I do not want to comment on something I do not understand well enough.

Reply:
Since this is an internal function, whatever goes is fine here :)